setup.py

```diff
@@ -9,7 +9,4 @@
     author='tumbler',
     author_email='zimbler@gmail.com',
     description='Vertica backend for Django',
-    install_requires=[
-        'vertica-python>=0.2.4,<0.3'
```

Comment:
This removal means that installed package won't work at all, because django_vertica_backend is a simple adapter between Django and vertica-python.